data/dev/analysis/filters/filter003/rubrics/_filters_rubric_template.yaml

```
@@ -0,0 +1,5 @@
+# Common rubric variables for filter003.
+topology_target: RL low-pass (first-order)
+topology_rejects: high-pass, notch, all-pass
+grounded_guidance: Cite R/L elements and nodes that form the filter path.
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)